jdaviz/configs/mosviz/mosviz.ipynb

```diff
@@ -36,7 +36,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.0"
+   "version": "3.10.4"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "f58cde014b621ad35d584ee345d351d3ac7f3f2b0ff1c403bd7acede792432eb"
```

Comment:
What is this?

Reply:
Ahh I opened up the notebook in my IDE. Let me strip that